src/components/User/KeyValue/KeyValueForm.vue

```diff
@@ -0,0 +1,267 @@
+        :disabled="formMixin.isDisabled"
+      >
+        <option
+          v-for="(value, lbl) in KEY_VALUE_ENTRY_TYPE"
+          :value="value"
+          :key="`update-entryType-${value}`"
+        >
+          {{ lbl }}
+        </option>
+      </select-field>
+    </div>
+
+    <div class="key-value-form__actions">
+      <form-confirmation
+        v-if="formMixin.isConfirmationShown"
+        :is-pending="isPending"
+        :message="confirmationMessage | globalize"
+        @ok="submit"
+        @cancel="closeConfirmation"
+        :ok-button-text="'key-value-form.btn-confirm' | globalize"
+        :cancel-button-text="'key-value-form.btn-cancel' | globalize"
+      />
+      <div
+        v-else
+        class="key-value-form__buttons"
+      >
+        <button
+          v-if="isUpdateForm"
+          class="app__btn-secondary"
+          type="button"
+          @click="closeModal"
+          :disabled="formMixin.isDisabled"
+
+        >
+          {{ 'key-value-form.btn-cancel' | globalize }}
+        </button>
+
+        <button
+          class="app__btn key-value-form__btn"
+          :disabled="formMixin.isDisabled"
+        >
+          {{ addButtonText | globalize }}
+        </button>
+
+        <button
+          v-if="isUpdateForm"
+          class="app__btn app__btn--danger key-value-form__btn"
+          type="button"
+          :disabled="formMixin.isDisabled"
+          @click="tryDeliteKeyValue">
+          {{ "key-value-form.btn-delete" | globalize }}
+        </button>
+      </div>
+    </div>
+  </form>
+</template>
+<script>
+
+import FormMixin from '@/mixins/form.mixin'
+
+import { required } from '@/validators'
+
+import { SelectField, InputField } from '@comcom/fields'
+import { base } from '@tokend/js-sdk'
+import { KEY_VALUE_ENTRY_TYPE } from '@/constants'
+import { api } from '@/api'
+import { ErrorHandler } from '@/utils/ErrorHandler'
+import { Bus } from '@/utils/bus'
+import { KeyValueRecord } from '@/js/records/keyValue.record'
+
+const EVENTS = {
+  closeModal: 'close-modal',
+  submited: 'submited',
+}
+
+export default {
+  components: {
+    SelectField,
+    InputField,
+  },
+
+  mixins: [FormMixin],
+
+  props: {
+    selectedKeyValue: {
+      type: KeyValueRecord,
+      default: () => new KeyValueRecord(),
+    },
+    isUpdateForm: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  data: _ => ({
+    form: {
+      key: '',
+      value: '',
+      entryType: KEY_VALUE_ENTRY_TYPE.uint32,
+    },
+    isPending: false,
+    isDeleteKeyValue: false,
+    KEY_VALUE_ENTRY_TYPE,
+  }),
+
+  validations () {
+    return {
+      form: {
+        key: { required },
+        value: { required },
+        entryType: { required },
+      },
+    }
+  },
+
+  computed: {
+    confirmationMessage () {
+      return this.isDeleteKeyValue ? 'key-value-form.msg-delete-these-key'
+        : 'key-value-form.msg-please-recheck-all-fields'
+    },
+
+    addButtonText () {
```

Comment:
addButtonTranslationId 
Give exact names to variables.
It will be text if you already translate

Reply:
fixed